libs/dao-data/src/utils/queries.ts

```diff
@@ -105,47 +101,77 @@ export const RAGE_QUIT_FIELDS = `
     id
     memberAddress
   }
-`;
+` as const;
 
-export const DAO_OVERVIEW = `
+export const DEFAULT_DAO_QUERY = `
   query dao($dao: String!) {
     dao(
       id: $dao
     ) {
-      ${DAO_FIELDS}
+      ${DEAFULT_DAO_FIELDS}
+      rageQuits {
+        ${DEFAULT_RAGE_QUIT_FIELDS}
+      }
     }
   }
-`;
+` as const;
 
-export const DAOS = `
+export const DEFAULT_DAOS_QUERY = `
   query daos {
     daos {
-      ${DAO_FIELDS}
+      ${DEAFULT_DAO_FIELDS}
+      rageQuits {
+        ${DEFAULT_RAGE_QUIT_FIELDS}
+      }
     }
   }
-`;
+` as const;
 
-export const DAO_PROPOSALS = `
+export const DEFAULT_PROPOSALS_BY_DAO_QUERY = `
   query proposals($dao: String!) {
     proposals(
       where: {dao: $dao}
       orderBy: createdAt
       orderDirection: desc 
     ) {
-      ${PROPOSAL_FIELDS}
+      ${DEFAULT_PROPOSAL_FIELDS}
+      votes {
+        ${DEFAULT_VOTE_FIELDS}
+      }
+    }
+  }
+` as const;
+
+export const DEFAULT_PROPOSAL_QUERY = `
+  query proposal($id: String!) {
+    proposal(id: $id) {
+      ${DEFAULT_PROPOSAL_FIELDS}
+      votes {
+        ${DEFAULT_VOTE_FIELDS}
+      }
+    }
+  }
+` as const;
+
+export const DEFAULT_MEMBERS_BY_DAO_QUERY = `
+  query members($dao: String!) {
+    members(
+      where: {dao: $dao}
+      orderBy: createdAt
+      orderDirection: desc 
+    ) {
+      ${DEFAULT_MEMBER_FIELDS}
     }
   }
-`;
+` as const;
 
-export const LATEST_TX = `
-  query eventTransaction {
-    eventTransactions(first: 1, 
-      orderBy: createdAt, orderDirection: desc) {
-      id
-      createdAt
+export const DEFAULT_MEMBER_QUERY = `
+  query member($id: String!) {
+    member(id: $id) {
+      ${DEFAULT_MEMBER_FIELDS}
```

Comment:
Going back to your comment in discord. We should definitely switch up the SDK as we have it setup now to rely on graphql code generation for all the types and even query utilities we need. The main would be it will be very easy to keep the sdk synced with the actual subgraph schema. Superfluid provides a good example of this and is how they have such a flexible api. This comment is more about creating a ticket in the future or hopefully giving a direction for some of the query stuff we are working on.

https://www.graphql-code-generator.com/

Reply:
> This comment is more about creating a ticket in the future or hopefully giving a direction for some of the query stuff we are working on.
> 
> https://www.graphql-code-generator.com/

Yeah this would be good to work on soon. That code generator tool looks nice.

I started looking aat packages to help generate the querys like this:
https://github.com/atulmy/gql-query-builder

Could be helpful/inspiration.